frontend/components/AuctionEventsBlock.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="mt-5">
+        <p class="mb-2">
+            The auction was taken via {{ takeEvents.length }} transaction<span v-if="takeEvents.length !== 1">s</span>
+            at {{ takeEvents[takeEvents.length - 1].transactionDate.toISOString() }}.
+        </p>
+        <ul class="list-disc list-inside">
+            <li v-for="(event, index) of takeEvents" :key="index">
```

Comment:
Providing something unique from the data itself is always better than array index:
```suggestion
            <li v-for="event of takeEvents" :key="event.transactionHash">
```